src/include/traffic_cop/traffic_cop.h

```diff
@@ -142,6 +142,7 @@ class TrafficCop {
   executor::ExecutionResult p_status_;
 
   void SetDefaultDatabaseName(std::string default_database_name) {
+    // check before changing the value!
```

Comment:
It looks like you didn't push the actual code. It's just a comment. Surely that's not supposed to fix any problem :)